massa-models/src/storage.rs

```diff
@@ -6,6 +6,7 @@ use std::sync::Arc;
 pub struct StoredBlock {
     pub block: Block,
     pub serialized: Vec<u8>,
+    pub serialized_header: Option<Vec<u8>>,
```

Comment:
Why is it an option  ?

Reply:
Currently we only compute it when sending out the header, and then cache it. So when first receiving the block it is set to `None`. Not sure that is the right way by the way.

Reply:
what happens when we first receive the header and then the block ?

Reply:
Currently nothing is done with individual headers. By the way I noticed we only propagate headers after having the received the full block.